msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -70,6 +76,19 @@
  * &lt;/activity&gt;
  * </pre>
  * </p>
+ * <p>Other Terminology:</p>
+ * <p>
+ *    <p><b>Scopes:</b>Permissions that the developers wants included in the access token received . Not all scopes are
+ *     guaranteed to be included in the access token returned.
+ *     </p>
+ *     <p>
+ *         <b>Login Hint:</b> Usually an email, to pass to the service at the beginning of the interactive authentication flow.
+ *     </p>
+ *     <p>
+ *         <b>Additional Scopes:</b>  Permissions you want the user to consent to in the same authentication flow,
```

Comment:
good point

---
In reply to: [114633613](https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/160#discussion_r114633613) [](ancestors = 114633613)